### PR TITLE
Add Offering Type column to Service Offering reference table

### PR DESCRIPTION
**Pull request by jasonburkert-ccpo**
- Adds `Offering Type` column to `Service Offering` reference table
- Updates reference data rows in table to include value for this new column
- Updates diagrams.md to reflect the change

Ticket: AT-8683

### CODE REVIEW

**Comment by jasonburkert-ccpo** [comment]
### New column

image

### Updated reference data to include value for new column

image

File: diagrams.md [hunk 1]
```diff
@@ -497,6 +497,7 @@ erDiagram
         Choice service_offering_group "ADVISORY_ASSISTANCE/APPLICATIONS/COMPUTE/DATABASE/DEVELOPER_TOOLS/DOCUMENTATION_SUPPORT/EDGE_COMPUTING/GENERAL_CLOUD_SUPPORT/GENERAL_XAAS/HELP_DESK_SERVICES/IOT/MACHINE_LEARNING/NETWORKING/PORTABILITY_PLAN/SECURITY/STORAGE/TRAINING"
         Integer sequence
         String name
+        Choice offering_type "XAAS_SERVICE/CLOUD_SUPPORT"
```
Comment (juliusfitzhugh-ccpo) [inline diagrams.md:500]: Not sure if `dow_task_number` was left out intentionally or not, but should that be included in the diagram for tables updated like `TRAINING_ESTIMATE`?

File: f600233d1b154d507b782f84604bcb12/update/sys_metadata_link_32343365974b11106fa8b4b3f153afde.xml [hunk 2]
```diff
@@ -6,29 +6,30 @@
     <x_g_dis_atat_service_offering action="INSERT_OR_UPDATE">
         <description/>
         <name>Application</name>
+        <offering_type>XAAS_SERVICE</offering_type>
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_metadata_link_32343365974b11106fa8b4b3f153afde.xml:9]: Kind of related to the XaaS services so I put it on this line. The `dow_task_number` was added in https://github.com/dod-ccpo/atat-snow/pull/307 for tables like `Training Estimate`, should a `dow_task_number` column also be added to the `Environment Instance` table to be extended to tables like `Compute Environment Instance` or was that left out intentionally to be handled a different way in the UI?

**Review by juliusfitzhugh-ccpo — COMMENTED** [review]
Looks good. Added a couple of questions.

File: diagrams.md (hunk 1 above)
Comment (jasonburkert-ccpo) [reply diagrams.md:500]: If I understand your question, I believe [this commit](https://github.com/dod-ccpo/atat-snow/pull/307/commits/38786895e3d0667a1d81487b1acc3ea5d1adb01f) added those columns to the documentation.
#307 should have all of the columns that front-end will need to store numbers per Tony.

File: f600233d1b154d507b782f84604bcb12/update/sys_metadata_link_32343365974b11106fa8b4b3f153afde.xml (hunk 2 above)
Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_metadata_link_32343365974b11106fa8b4b3f153afde.xml:9]: I believe that for environment instances, that information will be stored in `IGCE Estimate.dow_task_number` which pre-existed #307, but it does serve a similar purpose to the columns added in that PR.

File: diagrams.md (hunk 1 above)
Comment (juliusfitzhugh-ccpo) [reply diagrams.md:500]: Oh okay, I completely missed those in the documentation 😆

**Review by juliusfitzhugh-ccpo — APPROVED** [review]

**Pull request merged**